docs/code-snippets/word-snippets.yaml

```
@@ -1,3 +1,19 @@
+Word.Application#createDocument:member(1):
+  - |-
+    // Update the text of the current document using the text from another document passed in as base64.
+    await Word.run(async (context) => {
+        const externalDoc = context.application.createDocument(getBase64());
+        await context.sync();
+
+        const externalDocBody = externalDoc.body;
+        externalDocBody.load("text");
+        await context.sync();
+
+        const externalDocBodyText = externalDocBody.text;
+        const body = context.document.body;
+        body.insertText(externalDocBodyText, WordInsertLocation.start);
```

Consider this naming suggestion to help delineate between the two bodies.
```suggestion
        const currentDocBody = context.document.body;
        currentDocBody .insertText(externalDocBodyText, WordInsertLocation.start);
```